### DISCUSSION TITLE
inCondi ET660i support

### DISCUSSION THREAD

**Discussion by konrad-cz** [cagnulein/qdomyos-zwift#1002] [Q&A]
Good evening, 

I wanted to ask if the app is supporting the inCondi ET660i. The app did not find the device when I searched for it. I'd like to use it fo running. 
Thank you!
Kind regards,
O.

**Comment by konrad-cz** [comment]
![IMG_1455](https://user-images.githubusercontent.com/82611375/197625249-82bcc33d-95e8-4657-9b73-d04654f70278.PNG)
![IMG_1454](https://user-images.githubusercontent.com/82611375/197625254-9f5de4ae-264c-47ac-83a8-75cfb5f2985b.PNG)

**Reply by cagnulein** [reply]
hi @konrad-cz i can't see your bike in the nrfconnect screenshot. which one is it?

**Reply by konrad-cz** [reply]
Hi @cagnulein I was hoping you'd know as I could not tell :) Hopefully, I'll be able to find out tonight, I'll let you know. Thanks

**Reply by cagnulein** [reply]
Ehehe @konrad-cz you can check turning off and on the bike and checking which devices comes up and down

**Reply by konrad-cz** [reply]
Hi, I was able to find it. 
![WhatsApp Image 2022-10-25 at 22 22 48](https://user-images.githubusercontent.com/82611375/197875515-476f11fb-6ee5-4f32-a15b-a0ffa0e4572d.jpeg)
![IMG_1458](https://user-images.gith
![IMG_1459](https://user-images.githubusercontent.com/82611375/197875741-bbcd1548-7ee7-4f03-ae14-a6f7b967c34f.PNG)
ubusercontent.com/82611375/197875624-4524d401-742f-4f08-94ad-431f5b6e1f02.PNG)

**Reply by konrad-cz** [reply]
![IMG_1458 (1)](https://user-images.githubusercontent.com/82611375/197875903-8d6af6c9-0735-489e-bbd1-bc06d34fe845.PNG)

**Reply by konrad-cz** [reply]
Let me know if this is sufficient. Have a good evening.

**Reply by cagnulein** [reply]
@konrad-cz yes this seems right! Do you already use an app with this one?if so which is the name of the app?
Let me know 
Thanks

**Reply by konrad-cz** [reply]
@cagnulein the all is called mepanel, available on appstore. https://apps.apple.com/cz/app/mepanel/id863939051?l=cs

**Comment by cagnulein** [comment]
ok i created this issue for this https://github.com/cagnulein/qdomyos-zwift/issues/1007
